ocdata/utils/vasp.py

```diff
@@ -79,58 +85,60 @@ def _clean_up_inputs(atoms, vasp_flags):
         atoms       `ase.Atoms` object of the structure we want to relax, but
                     with the unit vectors fixed (if needed)
         vasp_flags  A modified version of the 'vasp_flags' argument
-    '''
+    """
     # Check that the unit vectors obey the right-hand rule, (X x Y points in
     # Z). If not, then flip the order of X and Y to enforce this so that VASP
     # is happy.
     if np.dot(np.cross(atoms.cell[0], atoms.cell[1]), atoms.cell[2]) < 0:
         atoms.set_cell(atoms.cell[[1, 0, 2], :])
 
     # Calculate and set the k points
-    if 'kpts' not in vasp_flags.keys():
+    if "kpts" not in vasp_flags.keys():
         k_pts = calculate_surface_k_points(atoms)
-        vasp_flags['kpts'] = k_pts
+        vasp_flags["kpts"] = k_pts
 
     return atoms, vasp_flags
 
 
 def calculate_surface_k_points(atoms):
-    '''
+    """
     For surface calculations, it's a good practice to calculate the k-point
     mesh given the unit cell size. We do that on-the-spot here.
 
     Arg:
         atoms   `ase.Atoms` object of the structure we want to relax
     Returns:
         k_pts   A 3-tuple of integers indicating the k-point mesh to use
-    '''
+    """
     cell = atoms.get_cell()
     order = np.inf
     a0 = np.linalg.norm(cell[0], ord=order)
     b0 = np.linalg.norm(cell[1], ord=order)
     multiplier = 40
-    k_pts = (max(1, int(round(multiplier/a0))),
-             max(1, int(round(multiplier/b0))),
-             1)
+    k_pts = (
+        max(1, int(round(multiplier / a0))),
+        max(1, int(round(multiplier / b0))),
+        1,
+    )
     return k_pts
 
 
-def _set_vasp_command(n_processors=16, vasp_executable='vasp_std'):
-    '''
+def _set_vasp_command(n_processors=16, vasp_executable="vasp_std"):
```

Comment:
This isn't used, you can remove it.

Reply:
Updated in https://github.com/Open-Catalyst-Project/Open-Catalyst-Dataset/commit/ac78be51e6108b9d69c071386844617d715ff4ab